assets/sass/styles.scss

```diff
@@ -3,7 +3,7 @@
 $backlog-colour: govuk-colour("purple");
 $discovery-colour: govuk-colour("bright-purple");
 $alpha-colour: govuk-colour("pink");
-$private-beta-colour: govuk-colour("brown");
+$private-beta-colour: #f7a074;
```

Comment:
@psoleckimoj  I was going to suggest _govuk-colour("light-pink")_ ... its seems fit if you look at the[ GDS colour palette. ](https://design-system.service.gov.uk/styles/colour/) ?

Reply:
@swilson-mojhub it's not really light orange though :/ The GDS does allow 25%/50% tints, it just doesn't specify them on that page with particular styles?

Reply:
ah I see... coolio :)